data/README.md

```diff
@@ -117,3 +122,17 @@ https://github.com/apache/parquet-format/commit/54839ad5e04314c944fed8aa4bc6cf15
 
 `bloom_filter.xxhash.bin` uses the newer xxHash-based bloom filter format as of
 https://github.com/apache/parquet-format/commit/3fb10e00c2204bf1c6cc91e094c59e84cefcee33.
+
+## NaN in stats
+
+Previous versions of the C++ Parquet writer would write NaN values in min and max
+statistics. It has been updated since to ignore NaN values when calculating
+statistics, but for backwards compatibility the following rules were established
+(in [PARQUET-1222](https://github.com/apache/parquet-format/pull/185)):
+
+> For backwards compatibility when reading files:
+> * If the min is a NaN, it should be ignored.
+> * If the max is a NaN, it should be ignored.
+> * If the min is +0, the row group may contain -0 values as well.
+> * If the max is -0, the row group may contain +0 values as well.
+> * When looking for NaN values, min and max should be ignored.
```

Comment:
I think it would be useful to add the "File was generated with: [...]" snippet that is part of the PR description.

Reply:
I was thinking it would always be in the PR history, but your are right that in the text would be more convenient.